apps/cyberstorm-storybook/package.json

```diff
@@ -41,6 +41,7 @@
     "@babel/preset-env": "^7.16.11",
     "@babel/preset-typescript": "^7.16.0",
     "@thunderstore/cyberstorm": "^0.1.0",
+    "@thunderstore/cyberstorm-styles": "^0.0.0",
```

Comment:
Nitpick: semver should start from 0.1.0.

Reply:
Fixed 👍